### ISSUE TITLE
Dashboard: low contrast

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#76]
There are a number of text elements with poor contrast on the dashboard:

> The contrast between the small text and background is not sufficient. The contrast ratio is 2.36:1 when it should be 4.5:1 for text smaller than 18px.

<img width="213" height="129" alt="Image" src="https://github.com/user-attachments/assets/c5cb5d6c-6488-442f-a8c0-8fe5aad33508" />

> The contrast between the small text and background is not sufficient. The contrast ratio is 2.36:1 when it should be 3:1 for bold text bigger than 14px.

<img width="213" height="115" alt="Image" src="https://github.com/user-attachments/assets/26442ce9-2dab-47ed-a2a7-bcae0ce79c4f" />

> The contrast between the small text and background is not sufficient. The contrast ratio is 3.61:1 when it should be 4.5:1 for text smaller than 18px.

<img width="213" height="92" alt="Image" src="https://github.com/user-attachments/assets/0e791bc8-9659-4811-abc3-f1588a5ebe1f" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
